src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/staticvariablename/Example2.java

```diff
@@ -0,0 +1,34 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="StaticVariableName">
+      <property name="applyToPublic" value="false"/>
+      <property name="applyToProtected" value="false"/>
+    </module>
+  </module>
+</module>
+*/
+
+
+package com.puppycrawl.tools.checkstyle.checks.naming.staticvariablename;
+
+// xdoc section -- start
+class Example2 {
+  public static int goodStatic = 2;
+  private static int badStatic = 2;
+  public static int GoodStatic1 = 2;
+  // ok above, because public variable is excluded in pattern
```

Comment:
`ok above, because public variable is not a target`

Or even better , remove comment completely. No comment means ok. You can leave empty line instead.

Reply:
Done, Updated with empty line